src/pages/chat-list/components/ChatListManageBar.tsx

```
@@ -0,0 +1,90 @@
+            </button>
+          );
+        })}
+      </div>
+      <div>
+        {isEditing ? (
+          <div className='flex gap-[1rem]'>
+            <ButtonTrash handleClick={handleDeleteChat} />
+            {selectChatList.size !== 0 ? (
+              <Button
+                style={{ paddingLeft: '0.75rem', paddingRight: '0.85rem' }}
+                children={'선택해제'}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)